msteams-platform/concepts/deploy-and-publish/apps-publish-overview.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Overview of publish your app
+description: This is an overview of how to package and publish your app.
+ms.topic: conceptual 
+keywords: teams apps Upload
+---
+
+# Publish your apps
+Apps that you develop need to be published to your organization for the users to use them. There is a complete process to publish your apps that includes, selecting methods of distribution, packaging and uploading, and finally publishing.
+
+This document provides an overview of the steps you must take to publish your app successfully to an App Store or AppSource, formerly Office Store.
+
+## Distribute your apps
+You can distribute your apps to an Enterprise organization, for example, a corporate or US Government organizations, such as federal, state, territorial, and tribal bodies. When distributing the apps, you must ensure that you adhere to the compliance policies of the selected organization. There are different methods to distribute your app. For more information, see [distribute your app](overview.md).
+
+## Create your app package
+After selecting the method to distribute your app, you need to create an app package. An app package helps you upload all core components of your app to the App Store or AppSource at one go. For more information, see [create your app package](../build-and-test/apps-package.md).
+
+## Upload your app package
+You can upload your app package to a team or a conversation through the App Store. You can also upload it using the AppSource. For more information, see [upload your app package](apps-upload.md).
+
+## Access your uploaded app 
+After uploading your app package, access it and test the components before publishing to your organization. For more information, see [access your uploaded app](apps-access.md).
```

Comment:
I think we can remove this and the related page. To me, this should be covered in the "Upload your app for a team or user"